h/paginator.py

```diff
@@ -53,7 +53,7 @@ def paginate(request, total, page_size=PAGE_SIZE):
     max_right = current_page + buffer
 
     i = current_page + 1
-    while i <= max_right and i > current_page and i < page_max:
+    while max_right >= i > current_page and i < page_max:
```

Comment:
I wonder if it's clearer if the items get bigger left to right?

```suggestion
    while current_page < i <= max_right and i < page_max:
```